tests/components/AddonList.test.js

```diff
@@ -125,11 +125,6 @@ describe('<AddonList />', () => {
         expect(searchedAddonslist.find("td")).to.have.length(10);
     });
 
-    it('should render div tags', () => {
-        expect(testAddonList.find("div")).to.have.length(6);
```

Comment:
Why are we getting rid of this?

Reply:
This is a bad test in that, it does not confirm the functionality of the application in the case of it being a unit test, or how the page would appear to the user in the case of it being a user acceptance test. To add to that on addition of external UI components would alter the number of div's that can not be counted. To conclude when testing a page it would be better to test static content that we expect to appear on the page, like a title probably in an h3 tag, or a table that is to contain the data etc, but for the case of div's these are not static as they keep changing whenever new changes or components are added to the page.